packages/react-import-remote/src/hooks.ts

```diff
@@ -103,9 +103,8 @@ export function useImportRemote<Imported = unknown>(
     () => {
       if (defer === DeferTiming.Idle) {
         if ('requestIdleCallback' in window) {
-          idleCallbackHandle.current = (window as WindowWithRequestIdleCallback).requestIdleCallback(
-            loadRemote,
-          );
+          idleCallbackHandle.current = (window as WindowWithRequestIdleCallback &
```

Comment:
nit: maybe we should add `type ExtendedWindow<T> = window & typeof globalThis & T` to `@shopify/useful-types` and use it for all these weird window extensiony things

Reply:
Added to useful-types